packages/extension-driver-duckdb/src/lib/duckdbExtensionLoader.ts

```diff
@@ -0,0 +1,74 @@
+import * as duckdb from 'duckdb';
+import { getLogger } from '@vulcan-sql/core';
+import { ConfigurationParameters } from './duckdbDataSource';
+
+// DuckDB parameter name : configuration name
+export enum HTTPFS_CONFIGURATIONS {
+  s3_region = 'region',
+  s3_access_key_id = 'accessKeyId',
+  s3_secret_access_key = 'secretAccessKey',
+  // sessionToken= alternative option for accessKeyId and secretAccessKey
+  s3_session_token = 'sessionToken',
+  s3_endpoint = 'endpoint',
+  s3_url_style = 'url_style',
+  s3_use_ssl = 'use_ssl',
+}
```

Comment:
Another suggestion~
I will recommend not using `enum` for configurations because it is not designed for the purpose
it could change to simply use an `object` to map instead, or also can do `freeze` to prevent value change if needed:

```ts
const HTTPFS_CONFIGURATIONS = Object.freeze({
  s3_region: 'region',
  s3_access_key_id: 'accessKeyId',
  ...
})
```